test/hotspot/jtreg/compiler/cha/AbstractRootMethod.java

```diff
@@ -38,6 +38,7 @@
  *                   -XX:CompileCommand=compileonly,*::test -XX:CompileCommand=dontinline,*::test
  *                   -Xbatch -Xmixed -XX:+WhiteBoxAPI
  *                   -XX:-TieredCompilation
+ *                   -XX:-StressMethodHandleLinkerInlining
```

Comment:
Wouldn't this fail with a product VM build because the flag is debug only?

Reply:
Adding `-XX:+IgnoreUnrecognizedVMOptions` to 3 files, in other cases it is already there, so if a develop flag is used on a product build, that flag would simply be ignored.